add bootnodes and static nodes content (#164)
ConsenSys/doc.goquorum: docs/HowTo/Configure/Bootnodes.md

[`--discovery.dns`](https://geth.ethereum.org/docs/interface/command-line-options) and
[`--port`](https://geth.ethereum.org/docs/interface/command-line-options) options.

## Production networks

A GoQuorum network must have at least one operating bootnode.
To allow for continuity in the event of failure, configure two or more bootnodes.

We don't recommend putting bootnodes behind a [load balancer](HighAvailability.md) because the
[enode URL](https://eth.wiki/en/fundamentals/enode-url-format) relates to the node public key, IP address, and discovery ports.
Any changes to a bootnode enode prevents other nodes from being able to establish a connection with the bootnode.
This is why we recommend putting more bootnodes on the network itself.

To ensure that a bootnode enode does not change when recovering from a complete bootnode failure:

1. Create the node key pair (the private and public key) before starting the bootnode.
1. When creating bootnodes in the cloud (for example, AWS or Azure), attempt to assign a static private IP address to
   them, and record this IP address.

We recommend that you store the bootnode configuration under source control.

> ### Specifying bootnodes
> 
> To allow for failure, specify all bootnodes in a comma-separated list using
> [`--bootnodes`](https://geth.ethereum.org/docs/interface/command-line-options) (even to the bootnodes themselves).
> 
> !!! example
> 
>     If your network has two bootnodes, pass the following parameter to all nodes, including the bootnodes.
> 
>     ```bash
>     --bootnodes=enode://<publicKeyBootnode1>@<ipBootnode1>:30303,<publicKeyBootnode2>@<ipBootnode2>:30303
>     ```
> 
> !!! tip
> 
>     Having each bootnode list the other bootnodes increases the speed of discovery.
>     Nodes ignore their own enode in the bootnodes list so it isn't required to specify different bootnode lists to the
>     bootnodes themselves.

Reviewer: ```suggestion
### Specifying multiple bootnodes

To allow for continuity in the event of failure, specify all bootnodes in a comma-separated list using the
[`--bootnodes`](https://geth.ethereum.org/docs/interface/command-line-options) option for each node, including the bootnodes. 

!!! example

    If your network has two bootnodes, pass the following parameter to all nodes, including the bootnodes.

    ```bash
    --bootnodes=enode://<publicKeyBootnode1>@<ipBootnode1>:30303,<publicKeyBootnode2>@<ipBootnode2>:30303
    ```

!!! tip

    Having each bootnode include the other bootnodes increases the speed of discovery.
    Nodes ignore their own enode in the bootnodes list so you can use the same bootnode list for all bootnodes.
```

> ### Adding and removing bootnodes

Reviewer: The title says removing bootnodes, but it doesn't actually cover it.  Is it as simple as taking them off the list or are there any other concerns?
